app/reducers/articleReducers.js

```diff
@@ -10,14 +12,37 @@ const initialState = {
   highlights: [],
 }
 
+function mergeHighlights(list) {
+  // TODO: write tests for me
```

Comment:
there's a bug I can't reproduce with this code, but it exists, I think. I'll look for it more when we have a testing plan